src/PieChart/PieArcSeries/PieArcSeries.tsx

```diff
@@ -50,47 +74,74 @@ export class PieArcSeries extends Component<PieArcSeriesProps> {
     };
   }
 
-  shouldDisplayLabel(arcData) {
+  shouldDisplayLabel(arcData: ArcData) {
     const { displayAllLabels } = this.props;
 
     return displayAllLabels || labelVisible(arcData);
   }
 
-  calculateLabelPositions(outerArc, outerRadius) {
-    const { label, data } = this.props;
+  calculateLabelPositions({
+    outerArc,
+    outerArcRadius,
+    minDistance
+  }: {
+    /**
```

Comment:
Why not pass them as individual arguments and document them in the docblock?

Reply:
I generally tend to use arguments when the number of arguments are <= 4. It makes it easier to read and understand. I hear your argument and usually do that for several props or props that are conditional combos ( such as overloads ).

Reply:
ok. I'll rewrite this